### ISSUE TITLE
Two SVGs on the same page interfering

### ISSUE THREAD

**Issue by EmptyCrown** [jhamlet/svg-react-loader#102]
In some cases, when I import two different SVGs onto the same page, their styles interfere, and one image becomes distorted (or there's an extra shadow, etc.).

For example:
```
import Tree from '-!svg-react-loader!../../images/tree.svg';
import Mountain from '-!svg-react-loader!../../images/mountain.svg';
```

The components are then rendered inline. Does anyone know how to make sure the SVGs don't interfere with each other?

**Comment by SilverFox70** [comment]
Yes! I had the same problem and the underlying cause has to do with the IDs in the SVGs. I have a created a fork of this that solves this problem and is very easy to use. One caveat: if you have two or more SVGs with the *same exact name* you will still get ID collisions and issues. Check it out here: https://github.com/SilverFox70/svg-react-loader

**Comment by jhamlet** [comment]
@SilverFox70 Do you want to contribute a pull request?

**Comment by SilverFox70** [comment]
Hi Jerry, I had meant to submit a PR a while ago but got sidetracked. Let me take a look to see if it needs any cleaning up/commenting and then I'll submit.

**Comment by Strangehill** [comment]
In trying to troubleshoot this issue, I reproduced the problem [here](https://svg-clashing-ids.vercel.app/)

Isn't the `classIdPrefix` query parameter mentioned in the README meant to address just this issue?

**Comment by 0xferit** [comment]
Any updates on this? I am having the same issue.
